component/file_cache/file_cache.go

```diff
@@ -69,7 +69,7 @@ type FileCache struct {
 	missedChmodList sync.Map
 	mountPath       string
 	allowOther      bool
-	directRead      bool
+	offloadIO       bool
```

Comment:
Should we just make this default behavior?

Reply:
yes its default only, offload here means offload to GO so default is false which means its native IO

Reply:
Why do we need this option at all?

Reply:
Just in case we want to disable this logic of doing read/write from libfuse. As not sure what kind of implication it will have on cache policy. So far we have not seen any issue but just in case in future we find some big hole there is a way to disable this feature.

Reply:
I think if the default behavior is on - its important to point this out in the README, not only in the baseConfig file. We should create a new file that contains critical things that customers should be aware of and this would be one of them.

Reply:
This is something totally internal, customer does not care what is our internal implementation for read/write or any optimizations. So, I don't any reason to document this in readme. For our own internal knowledge if we wish we can dump it in code comments or in some form of document. Again, this knob I am keeping just as a fail-safe, in case the CGO wrapper reading/writing file create any issue, otherwise code is tuned to take that path only by default.